checks/checks.md

```diff
@@ -23,6 +23,13 @@ This check tries to determine if a project has dependencies automatically update
 **Remediation steps**
 - Signup for automatic dependency updates with dependabot or renovatebot and place the config file in the locations that are recommended by these tools.
 
+## Binary-Artifacts 
+
+This check tries to determine if a project has binary artifacts in the source repository. These binaries could be compromised artifacts. 
```

Comment:
can add that building from source is recommended.

Reply:
I will include that. Thanks